site/en/docs/extensions/mv3/getstarted/tut-reading-time/index.md

```diff
@@ -0,0 +1,304 @@
+---
+layout: 'layouts/doc-post.njk'
+title: 'Reading time'
+description: 'Learn how to insert an element on each page.'
+subhead: 'Create your first extension that inserts a new element on the page.'
+date: 2022-08-10
+# updated: 2022-06-13
+---
+
+## Overview {: #overview }
+
+This tutorial builds an extension that adds the expected reading time to any Chrome extension and
+Chrome web store documentation page. 
+
+<figure>
+{% Img src="image/BhuKGJaIeLNPW9ehns59NfwqKxF2/VczSGe8eh0Xv7nTXxhxg.png", 
+alt="Reading time extension in the extension Welcome page", width="500", height="116", class="screenshot" %}
+  <figcaption>
+  Reading time extension on the extension's Welcome page. 
+  </figcaption>
+</figure>
+
+In this guide, we’re going to explain the following concepts:
+
+- The extension manifest.
+- What icon sizes an extension uses.
+- How to inject code on pages using [content scripts][doc-cs].
+- How to use match patterns.
+- Extension permissions
+
+## Before you start {: #prereq }
+
+This guide assumes that you have basic web development experience. We recommend checking out
+[Development Basics][doc-dev-basics] for an introduction to the extension development workflow.
+
+## Build the extension {: #build }
+
+To start, create a new directory called `Reading time` that will hold the extension's files. If you
+prefer, you can download the complete source code on [Github][github-reading-time].
+
+### Step 1: Add information about the extension {: #step-1 }
+
+The manifest JSON file is the only required extension file. It holds important information about the
+extension. Create a `manifest.json` file in the _root_ of the project and add the following code:
+
+{% Label %}manifest.json:{% endLabel %}
+
+```json
+{
+  "manifest_version": 3,
+  "name": "Reading time",
+  "version": "1.0",
+  "description": "Add the reading time to Chrome Extension documentation articles",
+  ...
+}
+```
+
+These keys contain basic metadata for the extension. It appears in the Extension Management page
+and, when published, in the Chrome Web Store listing. To dive deeper, check out the
+[`"name"`][man-name], [`"version"`][man-ver] and [`"description"`][man-desc] keys on the
+[Manifest][doc-manifest] overview page.
+
+{% Details %}
+{% DetailsSummary %}
+💡 **Other facts about the extension Manifest**
+{% endDetailsSummary %}
+
+- It must be located at the **root** of the project.
+- The only required keys are the `"manifest_version"`, `"name"` and `"version"`.
+- It supports comments (`//`) during development but not in the Chrome Web Store.
```

Comment:
For clarity, is this a packed/unpacked issue or does CWS not allow JSON files with comments? The distinction here is that Linux users can install self-signed extensions and managed environments can use enterprise policies to force-install self-signed extensions on all desktop platform.

Reply:
This happens when you upload your extension to the CWS developer dashboard. I took a screenshot of what happens when you try uploading a package with a manifest that includes comments:

<img width="585" alt="Manifest with comments are not allowed in the CWS" src="https://user-images.githubusercontent.com/37001393/186022385-10e3461b-08ff-4498-9b6f-28f49569026d.png">

This might not be the right context to include this information. Maybe it should go in the [Manifest file format](https://developer.chrome.com/docs/extensions/mv3/manifest/) doc?

Can you think of any other manifest file facts that devs should know? If not we can reconsider this detail section